Formula/librdkafka.rb

```diff
@@ -13,11 +13,12 @@ class Librdkafka < Formula
   end
 
   depends_on "lzlib"
+  depends_on "openssl"
 
   def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
-    system "make"
+    system "C_INCLUDE_PATH=.:#{include}:#{Formula["openssl"].opt_prefix}/include make"
```

Comment:
I tested this locally and it wasn't necessary. The `depends_on "openssl"` alone is sufficient to get it to use the brewed openssl instead of the system openssl.

Could you remove this line? Or if you're having trouble building without it, provide some details?